.env

```diff
@@ -42,5 +42,8 @@ MAIL_FROM_PASSWORD=
 # Cloudinary API URL to save images
 CLOUDINARY_URL=
 
-# Path to save file uploads, the path must be relative to the root of the project - Defaults to public
+# Hostname of the application (where the webserver will be served) - without the trailing '/' 
+WEBSERVER_HOST=https://localhost/8087
```

Comment:
After testing some functionality on another feature I noticed that this env. var. has the value "...localhost**/**8087". Shouldn't it be "...localhost**:**8087"?

Reply:
Definitely! Please make the change